sdk/sdk-api/src/main/java/mil/emp3/api/MilStdSymbol.java

```diff
@@ -1637,4 +1641,41 @@ public IEmpBoundingBox getFeatureBoundingBox() {
 
         return bBox;
     }
+
+    public void setIconColor(final IGeoColor color) {
```

Comment:
I know that these sets and gets are obvious on what they do, but if you could please add comments :) thank you!